packages/block-library/src/list-item/index.js

```diff
@@ -27,6 +29,7 @@ export const settings = {
 		};
 	},
 	transforms,
+	[ unlock( privateApis ).requiresWrapperOnCopy ]: true,
```

Comment:
Does this implicitly expose the block's new configuration? Or should we add new support to block.json, like `__unstablePasteTextInline`?

Reply:
For example, when I copied and pasted Button, Column, etc. that depend on the parent block, I found that the block could not be pasted like this problem. I imagine that similar problems can be solved by adding it as block support.

https://github.com/WordPress/gutenberg/assets/54422211/a596c483-4bfc-42bd-8b91-3624e6464f38

Reply:
> Does this implicitly expose the block's new configuration?

What do you mean? How is it exposed? You need the key to be able to retrieve it.

Sure, we could add it to column and button in the same way. I still prefer using a symbol until we properly stabilise it.

Reply:
For example, developers may discover and use this new key when creating custom blocks. Since it is a private API, is it okay not to take that into account?

Reply:
I see, then there's no problem. Maybe I didn't understand what a Symbol was 😅